doc/source/rllib/rllib-dev.rst

```diff
@@ -114,3 +116,30 @@ The objects with the top 20 memory usage in the workers will be added as custom
 
 .. image:: images/MemoryTrackingCallbacks.png
 
+
+Troubleshooting
+---------------
+
+If you encounter errors like
+`blas_thread_init: pthread_create: Resource temporarily unavailable` when using many workers,
+try setting ``OMP_NUM_THREADS=1``. Similarly, check configured system limits with
+`ulimit -a` for other resource limit errors.
+
+For debugging unexpected hangs or performance problems, you can run ``ray stack`` to dump
+the stack traces of all Ray workers on the current node, ``ray timeline`` to dump
+a timeline visualization of tasks to a file, and ``ray memory`` to list all object
+references in the cluster.
+
+TensorFlow 2.0
```

Comment:
Let's actually fix this now:
* "TensorFlow 2.x"
* "It is now recommended to use `framework=tf2` and `eager_tracing=True` (in case you are developing with TensorFlow) for maximum performance and support. We will however continue to support `framework=tf` (static-graph) for the foreseeable future.
* For debugging purposes, you should use `framework=tf2` + `eager_tracing=False`. All tf.Tensors' values will then be visible and printable when executing your code. However, some slowdown is to be expected in this config mode.